app/src/main/res/layout/credits_radio_button.xml

```diff
@@ -44,6 +45,52 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         />
+    <androidx.constraintlayout.widget.Group
+        android:id="@+id/credits_balances_group"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:constraint_referenced_ids="credits_fiat_balance,credits_text_separator,credits_balance_value"
+        />
+
+    <TextView
+        android:id="@+id/credits_fiat_balance"
+        style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="sans-serif-medium"
+        android:lineSpacingExtra="4sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="@id/title"
+        app:layout_constraintTop_toBottomOf="@id/title"
+        tools:text="31.90 EUR"
+        />
+    <TextView
+        android:id="@+id/credits_text_separator"
+        style="@style/TextAppearance.AppCompat.Body1.Roboto.Medium"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="|"
```

Comment:
This attribute should be defined as a untranslatable string in the resources folder